### PR TITLE
Typo and Link Corrections in the Documentation

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/4317

### CODE REVIEW

**Comment by SudharakaP** [comment]
@znarf : Who's the best person to assign for documentation related code reviews? 😄

**Review by znarf — APPROVED** [review]
Everything looks good. Feel free to merge.

**Comment by SudharakaP** [comment]
> Everything looks good. Feel free to merge.

@znarf : Thanks, but I'll add some more to this as I am still in progress reading the docs and I am sure I'll find many more. 😄

**Comment by SudharakaP** [comment]
@znarf : This is ready for review. Let me know if everything looks okay. 😄 This handles all simple issues I found; the more comprehensive list is given at, https://github.com/opencollective/opencollective/issues/4317.

**Review by znarf — CHANGES_REQUESTED** [review]
I generally prefer to link to https://opencollective.com/support instead of a mailto.

**Comment by SudharakaP** [comment]
> I generally prefer to link to https://opencollective.com/support instead of a mailto.

@znarf : Done. 😄 Let me know how it looks now. I've replaced almost all the mailto with the support link.

File: internal/welcome-document.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 description: >-
   Below you’ll find information to help you get started. Please read carefully &
   feel free to let us know if something isn’t clear enough or missing. You can
-  email Pia directly at pia@opencollective.co
+  email Pia directly at [pia@opencollective.co](mailto:pia@opencollective.co)
```
Comment (znarf) [inline internal/welcome-document.md]: .com

File: product/currencies.md [hunk 2]
```diff
@@ -13,5 +13,5 @@ Donation amounts will be denominated in the currency of the Collective's [Fiscal
 * NZD
 * JPY
```
Comment (znarf) [inline product/currencies.md:23]: Update with the currencies whitelisted in the frontend/api?

File: internal/welcome-document.md (hunk 1 above)
Comment (SudharakaP) [reply internal/welcome-document.md]: Ah, thanks for pointing it out. Corrected. 👍🏽

File: product/currencies.md (hunk 2 above)
Comment (SudharakaP) [reply product/currencies.md:23]: @znarf : Thanks, updated from, https://github.com/opencollective/opencollective-api/blob/d43ac68141bd9401091c6e67bf42361e8dab9fc7/server/constants/currencies.ts#L174-L206

**Comment by SudharakaP** [comment]
@znarf : Resolve some conflicts. Are we good to merge this? 😄 

cc: @Betree

**Review by Betree — APPROVED** [review]

**Pull request merged**